testapi.pm

```diff
@@ -307,7 +426,7 @@ within the block to avoid races between the action and the screen change
 
 =cut
 
-sub wait_screen_change(&@;$) {
+sub wait_screen_change(&@) {
```

Comment:
no, as I expressed - without the prototype you can't write 

```
wait_screen_change { send_key 'esc'; };
```

Only 
```
wait_screen_change sub { send_key 'esc' }; 
```
would work, but that's a bit too much perl